NEWS_DEV.txt

```diff
@@ -15,115 +15,147 @@ git shortlog -sn rel-0.8.0..
 
 TODO: better Theano conv doc
 
+# NB: Following notes are related to final release 0.9.0.
```

Comment:
All what is bellow seem a duplicate of what is in NEWS.txt. Also, what should be in NEWS_DEV.txt is what is in the master since the last release. So I would just remove all that.